### PR TITLE
Add microservice resources to no_proxy in the main ChatQnA README

### PR DESCRIPTION
**Pull request by daniel-de-leon-user293**
## Description

This PR only adds direction in the main ChatQnA README to append the microservice resources needed for proxy'd environments in the `no_proxy` env variable. These commands can be found in the individual hardware README's but just adding to the main README for more visibility.

## Issues

https://github.com/opea-project/GenAIExamples/issues/1247

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [x] Others (enhancement, documentation, validation, etc.)

## Dependencies

N/A

## Tests

N/A

### CODE REVIEW

File: ChatQnA/README.md [hunk 1]
```diff
@@ -60,15 +60,18 @@ To set up environment variables for deploying ChatQnA services, follow these ste
 
 3. Set up other environment variables:
 
-   > Notice that you can only choose **one** command below to set up envs according to your hardware. Other that the port numbers may be set incorrectly.
+   > Notice that you can only choose **one** hardware option below to set up envs according to your hardware. Other that the port numbers may be set incorrectly.
```
Comment (ashahba) [inline ChatQnA/README.md:63]: Do you mean:
`Other than` or `Other than that`?

**Review by ashahba — CHANGES_REQUESTED** [review]

File: ChatQnA/README.md (hunk 1 above)
Comment (ashahba) [reply ChatQnA/README.md:63]: I think the correct sentence should be:
`Also make sure port numbers are set correctly too.`

**Review by ashahba — APPROVED** [review]
LGTM!

**Review by yinghu5 — APPROVED** [review]
LGTM

**Pull request merged**